packages/contracts/contracts/foundry/SphinxUtils.sol

```diff
@@ -489,6 +489,26 @@ contract SphinxUtils is SphinxConstants, StdUtils {
         return config;
     }
 
+    function assertValidOwners(SphinxConfig memory _config) private pure {
+        require(
+            _config.owners.length > 0,
+            "Sphinx: You must have at least one owner in your 'sphinxConfig.owners' array before calling this function."
+        );
+
+        for (uint256 i = 0; i < _config.owners.length; i++) {
+            // We prevent the owners from being either address(0) or address(0x1)
+            // address(0x1) is the SENTINAL_ADDRESS which is a special address used
+            // by Safe to handle their owner implementation. We check for it here for
+            // completeness, but it's unlikely the user would ever actually use that
```

Comment:
I know it's unlikely that the user specifies `address(1)`, but i don't think it's unlikely enough to warrant throwing a misleading error message. e.g. maybe the user will do `address(0), address(1), address(2)` as placeholders for their real owner addresses

if you want to keep things concise, you could continue to use a single if-statement and error message by doing something like:
```
revert(abi.encodePacked("Sphinx: Detected owner that is, " _config.owners[i], ". You are not allowed to use this address as a Gnosis Safe owner"))
```

Reply:
FYI, your code suggestion doesn't work b/c encodePacked doesn't encode address(0) as a full address string (i.e 0x...00), it just treats it as an empty string.

Reply:
Ah yeah. Your change works. I think you could also do `vm.toString(_config.owners[i])`